js/actions/appActions.js

```diff
@@ -1198,6 +1198,67 @@ const appActions = {
       actionType: appConstants.APP_DRAGGED_OVER,
       draggedOverData
     })
+  },
+
+  /**
+   * Go back in a history for a given tab
+   * @param {number} tabId - Tab id used for an action
+   */
+  onNavigateBack: function (tabId) {
```

Comment:
just a nit, but it would be more consistent to use onGoBack, onGoForward, etc...